core/store/migrations/migration1560881852/migrate.go

```diff
@@ -0,0 +1,31 @@
+package migration1560881852
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+)
+
+func Migrate(tx *gorm.DB) error {
+	if err := tx.Exec(`ALTER TABLE job_specs ADD link_earned varchar(255)`).Error; err != nil {
+		return errors.Wrap(err, "failed to add a new link_earned column to job_specs table")
+	}
+	err := tx.Exec(`
+	UPDATE job_specs
+	SET link_earned = (
+	SELECT SUM(amount / 1000000000000000000) * 1000000000000000000 FROM
```

Comment:
I'm surprised division works on a `varchar` like this. `amount` is sql type varchar which is going to make`SUM` much harder, maybe.